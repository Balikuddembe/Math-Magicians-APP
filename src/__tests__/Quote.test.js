import React from 'react';
import { render } from '@testing-library/react';
import Quote from '../components/Quote';

describe('Quote component', () => {
  test('', () => {
    const { container } = render(
      <Quote />,
    );
    expect(container.firstChild).toMatchSnapshot();
  });
});
