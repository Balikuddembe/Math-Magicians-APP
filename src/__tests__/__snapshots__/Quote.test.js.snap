// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Quote component  1`] = `
<div
  class="quote_container"
>
  <p
    class="quote"
  >
    Mathematics is not about numbers, equations, computations, or algorithms: it is about understanding. -William Paul Thurston
  </p>
</div>
`;
